quick-starts/first-run/docs/Use-Neptune-API-to-log-your-first-run.ipynb

```diff
@@ -159,36 +195,42 @@
     "**Note**\n",
     "\n",
     "\n",
-    "Instead of logging data to the public project 'common/colab-test-run' as an anonymous user 'neptuner' you can log it to your own project.\n",
+    "Instead of logging data to the public project 'common/quickstarts' as an anonymous user 'neptuner' you can log it to your own project.\n",
```

Comment:
In the Neptune_API_tour NB you changed project to colab_test_run, while here to quickstart.

Pls, provide some reasoning around this change?
Preferably we want to log stuff to single project.

Reply:
This notebook shows the `quickstarts` project down below
<img width="518" alt="image" src="https://user-images.githubusercontent.com/41324509/167266330-6622b531-ec75-4438-93c1-a5ea9b8700ee.png">